src/components/Animaladd.js

```diff
@@ -1,35 +0,0 @@
-import React, { Component } from 'react';
-
-export default class Animaladd extends Component {
-    constructor() {
-        super()
-
-        this.state = {
-            name: '',
-            imageUrl: ''
-        }
-    }
-
-    updateNameInput = (name) => {
-        this.setState( {name} )
-    }
-
-    updateImgInput = (imageUrl) => {
-        this.setState( {imageUrl} )
-    }
-
-    addAnimalComp = () => {
-        this.props.addAnimal(this.state.name, this.state.imageUrl)
-        this.setState({ name: '', imageUrl: '' })
```

Comment:
One thing about this is if the animal doesn't add successfully, you've already blown away the inputs...not the worst thing in the world, but if you could chain it and _only_ set state if the call to `addAnimal` is successful that would be cool. in order for this to work you'd have to make sure to return the promise from `this.props.addAnimal`